Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetChannel.tsx

```diff
@@ -169,7 +169,7 @@ const AssetChannelWindow = (props: IProps) => {
                             }
                         }}
                         TheadStyle={{ fontSize: 'smaller', display: 'table', tableLayout: 'fixed', width: '100%' }}
-                        TbodyStyle={{ display: 'block', overflowY: 'scroll', maxHeight: window.innerHeight - 300, width: '100%' }}
+                        TbodyStyle={{ display: 'block', width: '100%' }}
```

Comment:
see https://github.com/GridProtectionAlliance/SystemCenter/blob/ProcessedFileFix/Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ProcessedFile/ByFile.tsx#L212

Reply:
I'll be closing this PR and reference this ticket in SC-101